app/scripts/modules/core/src/pipeline/filter/executionFilter.service.ts

```diff
@@ -11,25 +11,44 @@ import { FilterModelService, ISortFilter } from 'core/filterModel';
 import { Registry } from 'core/registry';
 
 const boundaries = [
-  { name: 'Today', after: () => moment().startOf('day') },
```

Comment:
is `Today` intentionally omitted here?

Reply:
I don't know what you're talking about.

Reply:
![nothing](https://media.giphy.com/media/26ufnwz3wDUli7GU0/giphy.gif)